lib/bundler/cli/exec.rb

```diff
@@ -70,6 +71,14 @@ def kernel_load(file, *args)
       abort "#{e.class}: #{e.message}\n  #{backtrace.join("\n  ")}"
     end
 
+    def process_title(file, *args)
+      if args.empty?
+        file
+      else
+        "#{file} #{args.join(" ")}".strip
```

Comment:
This new process title doesn't exactly match the old one, but it's enough for our purposes. I'm not super familiar with the conventions and behaviour of ruby processes, so I'm happy to modify/tweak it further if need be